accessibility_api/accessibility_lib/wrappers/IAccessible.py

```diff
@@ -0,0 +1,200 @@
+
+        json = {}
+        prefix = 'acc'
+
+        # Add field to show child is simple element
+        if DEBUG_ENABLED:
+            if child_id != CHILDID_SELF:
+                json['SimpleElement'] = True
+            else:
+                json['SimpleElement'] = False
+
+        for attribute in attributes:
+            field = attribute[len(prefix):]
+
+            if attribute in TREE_PROPERTIES:
+                if field == 'Children':
+                    if acc_ptr.accChildCount == 0:
+                        continue
+                    json[field] = getattr(self, 'get_' + field.lower())()\
+                        .get('Children')
+                else:
+                    json[field] = getattr(self, 'get_' + field.lower())()
+            else:
+                if callable(getattr(acc_ptr, attribute)):
+                    try:
+                        json[field] = getattr(acc_ptr, attribute)(child_id)
+                    except AttributeError:
+                        json[field] = 'Attribute Not Supported'
+                    except COMError:
+                        json[field] = 'Member Not Found'
+                else:
+                    json[field] = getattr(acc_ptr, attribute)
+
+        return json
+
+    def get_focus(self):
+        """
+        Get focused object
+        """
+
+        focus_val = getattr(self._target, 'accFocus')
+        return self.process_return(self._target, focus_val)
+
+    def get_selection(self):
+        """
+        Get selected object
+        """
+
+        select_val = getattr(self._target, 'accSelection')
+        return self.process_return(self._target, select_val)
+
+    def process_return(self, acc_ptr, value):
+        """
+        Handle multiple return types
+        """
+
+        if value is None:
+            return None
+        elif isinstance(value, int):
+            if acc_ptr in WinUtil.simple_elements:
+                return self.semantic_wrap(acc_ptr, value)
+            else:
+                return self.semantic_wrap(acc_ptr.accChild(value))
+        else:
+            return self.semantic_wrap(value)
+
+    def get_parent(self):
+        """
+        Get parent object
+        """
+
+        parent = getattr(self._target, 'accParent')
+        return self.semantic_wrap(parent)
+
+    def get_children(self):
+        """
+        Get children of accessible
+        """
+
+        tree = {}
+        return self.children(self._target, self.depth, tree)
+
+    def children(self, acc_ptr, child_depth, tree):
+        """
+        Get child accessible
+        """
+
+        # Check if there are children
+        if acc_ptr.accChildCount is 0:
+            return
+
+        # Determine depth of children
+        if (child_depth is not FULL_CHILD_TREE) and (child_depth is 0):
+            return
+        child_depth -= 1
+
+        # First is used to determine if a children field should wrap list
+        children_ptr = WinUtil._accessible_children(acc_ptr)
```

Comment:
get a pylint warning telling me _accessible_children is protected, i think you need to remove the _